packages/slider/src/Slider.ts

```diff
@@ -545,12 +546,13 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
     protected override willUpdate(changed: PropertyValues): void {
         if (changed.has('value') && changed.has('fillStart')) {
             this._cachedValue = Number(this.value);
-            if (this.fillStart) {
-                this.centerPoint = Number(this.fillStart);
-            } else {
+            // Test if fill-start is set without a value
+            if (this.getAttribute('fill-start') === '') {
```

Comment:
TBH I don't see any other option to check, if `fill-start` is provided, but without any value. As soon as I check on the property `fillStart` it converts both empty string and `0` to the number `0`, where I cannot decide, was it empty or really the value `0`.
Any ideas to improve?

Reply:
How about you check for the type of `fillStart`? It would have a value if it is not type of a `boolean`

Reply:
`typeof this.fillStart` always returns `number` and not `boolean` as lit maps that attribute always to a number property.